components/snackbar.html

```diff
@@ -0,0 +1,122 @@
+
+</head>
+
+<body>
+    <!-- navbar-section -->
+    <header>
+        <nav class="navigation">
+            <div class="nav-brand"><a class="link" href="/index.html">Rebalance-UI</a></div>
+            <ul class="non-bullet">
+                <li class="list-tem-inline"><a class="link link-active" href="/index.html">Home</a></li>
+                <li class="list-tem-inline"><a class="link" href="./avatar.html">Docs</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <!-- side-list bar -->
+    <div class="aside-container">
+        <aside class="aside-nav">
+            <ul class="list-item-non-bullet border-right">
+                <li class="navbar-vertical-item">
+                    <a class="aside-link" href="/introduction.html">Introduction</a>
+                </li>
+                <li class="navbar-vertical-item">
+                    <a class="aside-link" href="./avatar.html">Avatar</a>
+                </li>
+                <li class="navbar-vertical-item">
+                    <a class="aside-link" href="./alert.html">Alert</a>
+                </li>
+                <li class="navbar-vertical-item">
+                    <a class="aside-link" href="./badge.html">Badge</a>
+                </li>
+                <li class="navbar-vertical-item">
+                    <a class="aside-link" href="./button.html">Button</a>
+                </li>
+                <li class="navbar-vertical-item">
+                    <a class="aside-link" href="./card.html">Card</a>
+                </li>
+                <li class="navbar-vertical-item">
+                    <a class="aside-link" href="./card.html">Images</a>
+                </li>
+                <li class="navbar-vertical-item">
+                    <a class="aside-link aside-active-item" href="./input.html">Input</a>
+                </li>
+                <li class="navbar-vertical-item">
+                    <a class="aside-link" href="./list.html">Lists</a>
+                </li>
+                <li class="navbar-vertical-item">
+                    <a class="aside-link" href="./navigation.html">Navigation</a>
+                </li>
+                <li class="navbar-vertical-item">
+                    <a class="aside-link" href="./modal.html">Modal</a>
+                </li>
+                <li class="navbar-vertical-item">
+                    <a class="aside-link" href="./rating.html">Rating</a>
+                </li>
+                <li class="navbar-vertical-item">
+                    <a class="aside-link" href="./utilities.html">Text Utilities</a>
+                </li>
+                <li class="navbar-vertical-item">
+                    <a class="aside-link" href="./toast.html">Toast</a>
+                </li>
+            </ul>
+        </aside>
+
+
+        <!-- main-section -->
+        <main class="avatar-main">
+            <h1 class="avatar-heading">SnackBar</h1>
+            <!-- <p class="avatar-text">
```

Comment:
We can remove code comment if possible.

Reply:
yes, I will do but after two more components.